doc/luasnip.txt

```diff
@@ -1,75 +1,75 @@
+                                                                  \/_/ 
+<
 
 Luasnip is a snippet-engine written entirely in lua. It has some great
-features like inserting text (|luasnip-function-node|) or nodes
-(|luasnip-dynamic-node|) based on user input, parsing LSP syntax and switching
-nodes (|luasnip-choice-node|).
-
-All code-snippets in this help assume that >
-
-	local ls = require"luasnip"
-	local s = ls.snippet
-	local sn = ls.snippet_node
-	local t = ls.text_node
-	local i = ls.insert_node
-	local f = ls.function_node
-	local c = ls.choice_node
-	local d = ls.dynamic_node
+features like inserting text (`luasnip-function-node`) or nodes
+(`luasnip-dynamic-node`) based on user input, parsing LSP syntax and switching
+nodes (`luasnip-choice-node`).
+
+All code-snippets in this help assume that
+>
+    local ls = require"luasnip"
+    local s = ls.snippet
+    local sn = ls.snippet_node
+    local t = ls.text_node
+    local i = ls.insert_node
+    local f = ls.function_node
+    local c = ls.choice_node
+    local d = ls.dynamic_node
 <
-================================================================================
 
-SNIPPETS                                                         *luasnip-snippet*
+================================================================================
+SNIPPETS                                                        *luasnip-snippets*
 
-The most direct way to define snippets is `s`: 
+The most direct way to define snippets is `s`:
 >
-	s({trig="trigger"}, {})
+    s({trig="trigger"}, {})
+<
 
 (This snippet is useless beyond being a minimal example)
 
 `s` accepts, as the first argument, a table with the following possible
 entries:
 
-	`trig`        String      Trigger, Plain text by default    Required
-	`name`        String      Name, used by other plugins       Optional, `trig`
-	`dscr`        String      Description, used by other        Optional, `trig`
-	                        plugins
-	`wordTrig`    Boolean     If true, the snippet is only      Optional, `false`
-	                        expanded if the word (`[%w_]+`)
-	                        before the cursor matches the
-	                        trigger entirely
-	`regTrig`     Boolean     If true, `trig` is interpreted      Optional, `false`
-                            as a lua-pattern
+|------------|---------|--------------------------------|
+| Field      | Type    | Description                    |
+|------------|---------|--------------------------------|
+| `trig`     | String  |                                |
+| `name`     | String  | Name, used by other plugins    |
+| `dscr`     | String  | Description, used by other     |
+|            |         |                                |
+| `wordTrig` | Boolean | If true, the snippet is only   |
+|            |         |                                |
+| `regTrig`  | Boolean | If true, `trig` is interpreted |
+|------------|---------|--------------------------------|
```

Comment:
Could you fixup that table and insert the missing lines? (Was that formatted by stylua? Am I misreading the commit history?)